internal/persistence/postgres/postgres.go

```diff
@@ -129,6 +112,17 @@ func New(opts Opts, queryTimeout time.Duration, logger *zap.Logger) (persistence
 	return res, nil
 }
 
+// Driver implements the persistence.SQLPersister interface.
+func (s *Postgres) Driver() persistence.Driver { return persistence.Postgres }
+
+// SetDefaultSQLOptions implements the persistence.SQLPersister interface.
+func (s *Postgres) SetDefaultSQLOptions(db *sql.DB) error {
+	db.SetMaxOpenConns(persistence.DefaultMaxConn)
```

Comment:
In this file, you still have the old definition of these default values:

```
	defaultMaxConn         = 50
	defaultMaxIdleConn     = 20
	defaultMaxConnLifetime = time.Hour
	DefaultPort            = 5432
```

Reply:
Oops, nice catch!